internal/grpcutil/metadata.go

```diff
@@ -28,6 +28,11 @@ func ConvertMetadataToProtoHeader(
 ) []*v1.Header {
 	headerInfo := make([]*v1.Header, 0, len(src))
 	for key, value := range src {
+		// Omit grpc-status-details-bin from the response trailers reported
```

Comment:
This fixes the `client closed stdin` problem because the problem results from trying to write `grpc-status-details-bin` base64 encoded data to the output (not entirely sure why -- bc of utf-8? buffer length?) to stdout. But is this the right solution? Doing this removes anything grpc specific from the headers returned to the test. But maybe that's fine bc connect has already tried to parse the grpc stuff by now, so as long as we know the error that grpc returned by now we can discard anything we used to glean that information.

We sort of have to do this here bc it needs to be omitted before we send out the response, but thats probably
ok as long as we know the status and details by now.

Reply:
This is very confusing. I don't think we should have to do this, and it would be even more confusing if we suggested that other clients-under-test or server-under-test should also do this.

I think we need to identify the real root cause of _why_ this was causing the problem and (hopefully) resolve in a different way.

Reply:
Having said that, I don't think we need to block this PR. Let's add a TODO to investigate further and remove this when the issue is more clearly understood.